MaterialDesignThemes.Wpf/DataGridAssist.cs

```diff
@@ -339,6 +339,12 @@ public static void SetCornerRadius(DataGrid element, CornerRadius value)
         => element.SetValue(CornerRadiusProperty, value);
     #endregion
 
+    #region AttachedProperty : ElementStringFormatProperty
+    public static readonly DependencyProperty ElementStringFormatProperty = DependencyProperty.RegisterAttached("ElementStringFormat", typeof(string), typeof(DataGridAssist), new PropertyMetadata(null));
+    public static string? GetElementStringFormat(System.Windows.Controls.DataGridTextColumn element) => element.GetValue(ElementStringFormatProperty) is string s ? s : null;
+    public static void SetElementStringFormat(System.Windows.Controls.DataGridTextColumn element, string? value) => element.SetValue(ElementStringFormatProperty, value);
```

Comment:
This setter/getter pair seem to indicate that this works for normal `System.Windows.Controls.DataGridTextColumn` columns, however I don't believe that is the case because there you don't have control over the `GenerateElement()` method which you override in the MDIX variant of the `DataGridTextColumn` (file below).

Reply:
Good call. I originally thought this could apply to any DataGridTextColumn, but I left that thought halfway finished in the first commit.